camera_system/block_to_intersection.py

```diff
@@ -48,8 +48,10 @@ def generate_command(self) -> str:
 
         if self.__rotation_angle != 0:  # 回頭角度が0の場合は回頭のコマンドを生成しない
             # 回頭角度が正の数の場合時計回り，負の数の場合反時計回りで回頭をセットする
+            command_list += "SL,100\n"
```

Comment:
100msは定数みたいな感じで定義してた方がいいと思う
あと回頭の前後にスリープを入れる意図をコメントに書いてほしい
```suggestion
            command_list += "SL,%d\n" % GameMotion.SLEEP_TIME
```